### DISCUSSION TITLE
PhalconPHP BE dev wanted

### DISCUSSION THREAD

**Discussion by StevenFlecha** [phalcon/cphalcon#15676] [Jobs]
Sick of commuting and wasting hours of your day?
Tired of having to dress up (or dress at all :) ) every day to go to the office?
Want the freedom to work from any place in any country and make your own work hours?

Imagine yourself working for a cool online company (+/- 25 great team members) creating new cutting-edge products on the web. We share ideas and new programming techniques via our team chat with constant communication. We are easygoing but with a great work ethic and (most importantly) get shit done when the pressure is on.

Who we are and our mission
We are a young startup with a mission to facilitate and accelerate a global shift from a knowledge economy to an entrepreneurial economy. We believe in an abundance mindset and our software is here to help everyday people to forget about the technical distractions and focus on their unique superpowers and sell those online to help their audience improve their lives.

Do you want to grow together and help us move towards our big ambitions? Get attractive bonuses based on your work results? In return, we will, of course, respect your local holidays and you can work from any location you want.

Then keep reading!

This is just 100% you! - We Love Personality!

- Attention to detail, down to every tag, loop, and condition
- Driven by optimization and code quality
- Analytical mind that can turn every idea into a clean coded functionality
- You are reliable, proactive and you can focus on the task at hand and be able to meet deadlines
- You are fluent in English and can communicate your thoughts and ideas clearly
- You have a stable high-speed internet connection, can work from home (or your office / co-working, etc.) without distractions, and be available for calls and in our team chat
- Love for your craft

These are the skills we are looking for:
- A PHP wizard with at least 5 years of PHP programming experience under your belt
- A PhalconPHP expert with at least 1.5 years full-time experience with PhalconPHP -production- application(s)
- Experience working on API's (so creating API's)
- High attention to security; know your security prerequisites - these are the products we are working on, and we need them secured 100%
- GIT - to be able to work with Git Repos

Any of these skills make you go to the top of the list:
- Docker experience
- Payment processor deep integrations (Stripe, Paypal, Chargebee, Clickbank, Paddle, etc)
- MicroService experience
- Big data experience
- Data Modelling knowledge
- GraphQL experience
- Laravel experience
- Kubernetes experience

So be sure to mention if you have any of these!

About us - Kyvio
We are a startup with a fully remote team of 25 employees (+1?), and our primary business focus is our SaaS.  Kyvio is a marketing automation tool for online businesses and entails a funnel builder, membership site builder, (digital) product delivery and selling module, and an online customer journey builder.

This is a fully remote, full-time position in a team working in different parts of the world. The team is run by a CEO and CTO, both of whom are Digital Nomads working from different spots in the world! It doesn’t matter in which time zone you are currently based in, as long as you make it to our meetings that are a must to attend.

We look for dedication and professionalism while giving you creative freedom. We all contribute immensely to the success of our company, and all employees are an integral part of our success. We want people who believe in our product. We return, we give you the flexibility to work in your most motivating environment.

We love our multi-cultural company and the fact we have people from all over the world working on creating the best products to give our customers.

How to apply?
https://kyvio.vervoe.net/job-page/php-phalcon/c7c85843-fdc9-4eb3-ba37-5cddc9905173 

Salary
Your typical work week would be 40 hours.
The salary is negotiable based on the skill set but in the range of $15 - $25 USD per hour depending on your experience and skills.

We have a bonus and salary increase plan based on your responsibility in individual projects and the success of those projects. They are both also based on the results of the company - Kyvio’s success is your success as well!

We love ambition so if you want to improve your skills we offer you the room to do so and grow in the company.

Now - go on and apply already! :)

Sincerely,
Steven

**Comment by someson** [comment]
let's be honest, with all those required experiences i wouldn't even think about unless the price range starts at 50 USD per hour...

**Reply by StevenFlecha** [reply]
if you are from the USA living in California yes probably - there's many other places in the world where with the offered salary you can live like an absolute king.

But 'all those required experiences' ?? We are not asking that much. These are the requirements:

1. A PHP wizard with at least 5 years of PHP programming experience under your belt
2. A PhalconPHP expert with at least 1.5 years full-time experience with PhalconPHP -production- application(s)
3. Experience working on API's (so creating API's)
4. High attention to security; know your security prerequisites - these are the products we are working on, and we need them secured 100%
5. GIT - to be able to work with Git Repos

This is a mid-level developer. 

The rest (above that) are personality traits and bellow it are EXTRAs that make you go on top of the list - not requirements.

**Reply by wurst-hans** [reply]
I agree with @someson
I'm from Germany and you won't find a freelancer working for 13-21 EUR. Maybe in other regions of the world you'll find any developer for this salary. But not in Europe or USA (at least that's my opinion).
